System.NullReferenceException: IsTrue encountered NullReferenceException

l.ToUpperInvariant() == "Oops"
. .              .   __
. \______ _______/   |
.        |           |
|        |           (Threw NullReferenceException: Object reference not set to an instance of an object.)
|        (Threw NullReferenceException: Object reference not set to an instance of an object.)
null


 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at lambda_method(Closure )
   at PowerAssert.PAssert.IsTrue(Expression`1 expression) in ...\PAssert.cs
   --- End of inner exception stack trace ---
   at PowerAssert.PAssert.IsTrue(Expression`1 expression) in ...\PAssert.cs
   at PowerAssertTests.Approvals.EndToEndTest.ApproveException(Expression`1 func) in ...\EndToEndTest.cs